packages/pypy/build.sh

```diff
@@ -0,0 +1,278 @@
+TERMUX_PKG_HOMEPAGE=https://pypy.org
+TERMUX_PKG_DESCRIPTION="A fast, compliant alternative implementation of Python"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+_MAJOR_VERSION=2.7
+TERMUX_PKG_VERSION=7.3.6
```

Comment:
Have you tried 7.3.8? That version seem to [support openssl3](https://foss.heptapod.net/pypy/pypy/-/issues/3560), so that we can drop the openssl-1.1 dependency and patch

Reply:
I've just tried 7.3.8 and it can be built successfully. We can update it.